httpie/cli/definition.py

```diff
@@ -415,13 +415,13 @@
 )
 output_options.add_argument(
     '--output', '-o',
-    type=FileType('a+b'),
-    dest='output_file',
+    type=str,
```

Comment:
Did you look in to making a new type for this? Did that not turn out to be feasible?

Reply:
I think that checking for a valid string is all that might be needed here. That's because if the user enters any given string, it will create a download file with that given name. So I think the only check we need to make is for a valid string. (Let me know if you have differing viewpoints)

Reply:
Like Aksh said, the current --output allows the creation of file with the string that's passed in, so not much needs to be changed now that we've just changed the argument type to string.